test/context/cookies.js

```diff
@@ -40,13 +31,12 @@ describe('ctx.cookies.set()', () => {
           }
         });
 
-        request(app.listen())
-          .get('/')
-          .expect('.keys required for signed cookies', done);
+        const text = await (await request(app, '/')).text();
```

Comment:
lol. I always try to only do 1 async action per line. gets crazy like this otherwise